src/main/java/bio/terra/service/StudyService.java

```diff
@@ -62,10 +62,13 @@ public StudySummaryModel createStudy(StudyRequestModel studyRequest, Authenticat
         return getResponse(flightId, StudySummaryModel.class);
     }
 
-    public StudyModel retrieve(UUID id) {
+    public StudyModel retrieveModel(UUID id) {
```

Comment:
Dup of what is in DR-466. Do you need to create that collision?

Reply:
I needed to, but will rebase now that 466 is merged in.